ddtrace/llmobs/_trace_processor.py

```diff
@@ -50,6 +51,7 @@ def process_trace(self, trace: List[Span]) -> Optional[List[Span]]:
         if not trace:
             return None
         for span in trace:
+            record_span_finished(span)
```

Comment:
We need to make sure this line is inside the `span.span_type == SpanTypes.LLM` check since this technically iterates over all APM spans by the tracer.

Reply:
makes sense. fixed.